internal/cli/cmd/get/freight.go

```diff
@@ -108,6 +112,11 @@ func (o *getFreightOptions) addFlags(cmd *cobra.Command) {
 	)
 	option.Names(cmd.Flags(), &o.Names, "The name of a piece of freight to get.")
 	option.Aliases(cmd.Flags(), &o.Aliases, "The alias of a piece of freight to get.")
+	option.Origins(cmd.Flags(), &o.Origins, "The origin of the freight to get.")
```

Comment:
I am iffy on this. How do we with the `--origin` flag that we're looking from Freight from a Warehouse named `foo` vs (looking to the future) some other kind of origin (Junction?) named `foo`?

I am wondering if something like `--warehouse foo` (and later `--junction foo`) might be more expressive.

Reply:
I think it would change to e.g. `Warehouse/foo` to match the identifier we use elsewhere.

If the origin does not contain a `/`, we assume it's a Warehouse.

Reply:
I think thus far we've only used the form `Warehouse/foo` for display purposes, but I am ok with what you propose with the caveat that we revisit it if it proves to be a pain point after additional origin types are a reality.